vulture_allowlist.py

```diff
@@ -171,6 +171,7 @@
 MONKEY_LINUX_RUNNING  # unused variable (monkey/monkey_island/cc/services/utils/node_states.py:26)
 import_status  # monkey_island\cc\resources\configuration_import.py:19
 config_schema  # monkey_island\cc\resources\configuration_import.py:25
+get_files_to_encrypt  # monkey/infection_monkey/ransomware/utils.py:82
```

Comment:
Needs to be removed.

Reply:
It's ironic that vulture doesn't notice "dead code" in it's own whitelist file.